README.md

```diff
@@ -35,6 +35,22 @@ df.tensorflow.to_tfr(output_dir='gs://my/bucket')
 
 ##### Running on Cloud Dataflow
 
+Google Cloud Platform Dataflow workers need to be supplied with the tfrecorder
+package that you would like to run remotely.  To do so first download or build
+the package (a python wheel file) and then specify the path the the file when
+tfrecorder is called.
+
+Step 1: Download or create the wheel file.
+
+To download the wheel from pip:
+`pip download tfrecorder --no-deps`
```

Comment:
nonblock: Having the user download a wheel file to run TFRecorder on GCP seems cumbersome. I have no alternative ideas in mind at the moment, but we should come up with a better way.

Reply:
I totally agree.  I spent the weekend trying to come up with a better option.  Now that you're back I'll setup a chat to figure out what we can do.